src/components/topNav/primaryTopNav/PrimaryTopNav.module.css

```diff
@@ -1,9 +1,25 @@
 :root {
   --conduction-primary-top-nav-item-padding: var(--skeleton-size-md);
   --conduction-primary-top-nav-color: var(--skeleton-color-white);
-  --conduction-primary-top-nav-background-color: var(--skeleton-color-secondary-3);
-  --conduction-primary-top-nav-background-color-hover: var(--skeleton-color-secondary-5);
-  --conduction-primary-top-nav-dropdown-border-radius: var(--skeleton-border-radius-md);
+  --conduction-primary-top-nav-background-color: var(
+    --skeleton-color-secondary-3
+  );
```

Comment:
**This comment goes for all your files.**

Your `prettier` is not working correctly. The highlighted piece of code above:

```
  --conduction-primary-top-nav-background-color: var(
    --skeleton-color-secondary-3
  );
```
is actually rendering spaces, making the tokens not work (see screenshot).

Within the component package (above code, and some other rules within these files as well):
![image](https://user-images.githubusercontent.com/66552932/193207121-18039bf0-ebc2-4b35-9dc4-e5bffd80eb45.png)

Afterwards, the same is happening in the implementation (Open Catalogi):

![image](https://user-images.githubusercontent.com/66552932/193207280-f2510508-98b5-4949-8ec3-c8862ce48e63.png)

We have a file called `.prettierrc` within this (and all other) codebase(s), which creates generic rules for all developers to use. Something's not working as intended there, please do check before merging this pull request and the pull request implementing these changes.